### PR TITLE
Merge latest to production

### PR DESCRIPTION
**Pull request by burtonquantil**
including release notes and new FAQ

### CODE REVIEW

File: zh/docs/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## November 11, 2022
+### API updates
+* Fixed an error in the response of the [edge hostname list API](</apidocs#operation/getEdgeHostnames>).
+* Improved validation of origin server settings when 127.0.0.1 or localhost is used as a server to disallow use of nonpublic ports.
+* Improved validation of the dpaAgreedBy field when [updating a customer](</apidocs#operation/patch-ngadmin-customers-id>).
+* Improved validation of the report APIs’ filters field in the request body.
```
Comment (marcfishelquantil) [inline zh/docs/release-notes.md:8]: If we are referring to a single API, this should be "API's". The way it is written, this refers to multiple APIs.

File: en/docs/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # Release Notes
 
+## November 11, 2022
+### API updates
+* Fixed an error in the response of the [edge hostname list API](</apidocs#operation/getEdgeHostnames>).
+* Improved validation of origin server settings when 127.0.0.1 or localhost is used as a server to disallow use of nonpublic ports.
+* Improved validation of the dpaAgreedBy field when [updating a customer](</apidocs#operation/patch-ngadmin-customers-id>).
+* Improved validation of the report APIs’ filters field in the request body.
```
Comment (marcfishelquantil) [inline en/docs/release-notes.md:8]: If we are referring to a single API, this should be "API's". The way it is written, this refers to multiple APIs.

**Review by marcfishelquantil — COMMENTED** [review]
Please see comments.

File: en/docs/release-notes.md (hunk 2 above)
Comment (burtonquantil) [reply en/docs/release-notes.md:8]: @marcfishelquantil  that is intentional as we do have multiple report APIs.

**Review by marcfishelquantil — APPROVED** [review]
Approved

**Pull request merged**